pyproject.toml

```diff
@@ -4,15 +4,15 @@ version = "0.1.0"
 description = ""
 authors = []
 license = "MIT"
+#package.mode = false
 
 [tool.poetry.dependencies]
 python = "^3.10"
 pip = "^22.0.4"
 mavsdk = "*"
 numpy = "^1.21.2"
-opencv-python = "^4.5.3.56"
+opencv-python = "^4.8.0."
```

Comment:
Harper listed the version as 4.8.0.76 in the Discord, is is for future-proofing the poetry dependency?